CMakeLists.txt

```diff
@@ -91,6 +92,7 @@ message(STATUS "Enable with code coverage:          ${CONTOUR_CODE_COVERAGE_ENAB
 message(STATUS "Build contour frontend GUI:         ${CONTOUR_FRONTEND_GUI}")
 message(STATUS "Build contour using Qt 6:           ${CONTOUR_BUILD_WITH_QT6}")
 message(STATUS "Build contour using mimalloc:       ${CONTOUR_BUILD_WITH_MIMALLOC}")
+message(STATUS "Buildin FreeType and HarfBuzz:      ${CONTOUR_BUILD_WITH_HARFBUZZ}")
```

Comment:
This is a little confusing. Contour if already using freetype and harfbuzz. 
So the steering `CONTOUR_BUILD_WITH_HARFBUZZ` reads as if you can also compile this project without using harfbuzz.

In the other hand: what is the benefit of ending the harfbuzz/freetype source code?

Reply:
> This is a little confusing.

What about "Embedded FreeType and HarfBuzz"?

> what is the benefit of ending the harfbuzz/freetype source code?

Performance. I don't know what options were used to compile the system libraries.
I think `march=native` might improve something.

Reply:
probably more something like "Build contour with system harfbuzz/freetype" `CONTOUR_USE_SYSTEM_HARFBUZZ` defaulting to `ON`, and if off, it'll use the embedded version to be statically linked.

System libraries are never built with `-march=native` as this is not portable. for local builds, sure. I mean, it used to be heavily buggy (just like O3), but my experience on that one is also 20 years old, I'm sure it has changed since then. :)

I am however also not sure about the place you've put this. contour itself doesn't care about freetype/harfbuzz, but the `terminal_renderer` library does (check `src/terminal_renderer/CMakeLists.txt`). So this should be moved then to that place, too.

Reply:
p.s.: to keep it in line with the other messages, above, maybe have indeed the string like "Build contour using embedded harfbuzz & freetype" (then defaulting to off of course). :)